ModularTegustation/tegu_items/refinery/crates/corporation.dm

```diff
@@ -63,7 +63,10 @@
 		/obj/item/toy/plush/bongy,
 	)
 
-
+	cosmeticloot = list(
+		/obj/item/toy/plush/bongy,
+		/obj/item/clothing/suit/armor/ego_gear/city/kcorp_sci
```

Comment:
Please remember to put commas even if the object is at the end of the list, this helps if anyone wants to add onto the list in the future (since github will show it as 1 addition instead of 2 additions and 1 deletion)
```suggestion
		/obj/item/clothing/suit/armor/ego_gear/city/kcorp_sci,
```